package.json

```diff
@@ -127,22 +127,22 @@
     "@types/react-beautiful-dnd": "^13.1.3",
     "@types/react-dom": "^18.0.11",
     "@types/react-portal": "^4.0.4",
-    "axe-core": "4.4.3",
-    "chromatic": "^6.17.0",
+    "axe-core": "4.6.3",
```

Comment:
`^4.6.3`? 
Iirc we kept it at 4.4.3 due to timeouts but it looks like we're just skipping that. Can we remove `axe-core` from the ignore in the [renovate config](https://github.com/chanzuckerberg/edu-design-system/blob/main/.github/renovate.json5#L24-L25)?

Reply:
👍

Reply:
Can we add the caret back (shoulda been more clear lol sorry)